dali/pipeline/operator/builtin/external_source.cc

```diff
@@ -98,6 +98,12 @@ The operator will validate that the fetched data is of the provided type.
 If the argument is omitted or ``DALIDataType.NO_TYPE`` is passed, the operator will infer
 the type based on the provided data.
 
-This argument will be required starting from DALI 2.0.)code", DALI_NO_TYPE);
+This argument will be required starting from DALI 2.0.)code", DALI_NO_TYPE)
+  .AddOptionalArg<int>("ndim", R"code(Input dimensionality.
```

Comment:
IMHO, this is way more understandable ;) Your call though.
```suggestion
  .AddOptionalArg<int>("ndim", R"code(Number of dimensions in the input.
```

Reply:
done